picard/ui/metadatabox.py

```diff
@@ -143,6 +143,36 @@ def tag_status(self, tag):
         return TagStatus.NoChange
 
 
+class PreservedTags:
+
+    opt_name = 'preserved_tags'
+
+    def __init__(self):
+        self._tags = self._from_config()
+
+    def _to_config(self):
+        config.setting[self.opt_name] =  ", ".join(sorted(self._tags))
+
+    def _from_config(self):
+        return set(
+            [tag for tag in map(lambda x: x.strip(),
```

Comment:
Not a really big fan of this indentation :P

Reply:
You can replace lambda exp. with `str.strip`

Reply:
filter(bool, map(str.strip, config.setting[self.opt_name].split(','))) is prolly what  you want?

Reply:
See 7b94c79affb06b1d01bf28dfb649963751cbad13